clustering/faces/mojarra/src/main/java/org/wildfly/clustering/faces/mojarra/context/ContextImmutability.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.clustering.faces.mojarra.context;
+
+import java.util.Enumeration;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
+
+import jakarta.servlet.ServletContext;
+import jakarta.servlet.http.HttpSession;
```

Comment:
Since I'm on an import nazi kick....

jakarta isn't special like java and javax